docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_privatelink/_index.md

```diff
@@ -0,0 +1,245 @@
+    --name <The name of this deployment> \
+    --template-uri https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_arc_servers_jumpstart/azure/privatelink/azuredeploy.json \
+    --parameters <The *azuredeploy.parameters.json* parameters file location>
+    ```
+
+    For example:
+
+    ```shell
+    az deployment group create \
+    --resource-group Arc-Azure-PL-demo \
+    --name arcpldemo \
+    --template-uri https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_arc_servers_jumpstart/azure/privatelink/azuredeploy.json \
+    --parameters azuredeploy.parameters.json
+    ```
+
+     > **Note: The deployment may take around 20-30 minutes to complete**
+
+- Verify the resources are created on the Azure Portal.
+
+    ![Resources created on Azure's resource group](./02.png)
+
+- Once the deployment is done, create resources in the on-premises resource group. To deploy the ARM template, navigate to the local cloned [deployment folder](https://github.com/microsoft/azure_arc/tree/main/azure_arc_servers_jumpstart/privatelink) and run the below command:
+
+    ```shell
+    az deployment group create \
+    --resource-group <Name of the on-premise resource group> \
+    --name <The name of this deployment> \
+    --template-uri https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_arc_servers_jumpstart/azure/privatelink/onpremise.json \
+    --parameters <The *azuredeploy.parameters.json* parameters file location>
+    ```
+
+    For example:
+
+    ```shell
+    az deployment group create \
+    --resource-group Arc-Onpremise-PL-Demo \
+    --name arcpldemo \
+    --template-uri https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_arc_servers_jumpstart/azure/privatelink/onpremise.json \
+    --parameters onpremisedeploy.parameters.json
+    ```
+
+     > **Note: The deployment may take around 20-30 minutes to complete**
+
+- Verify the resources are created on the Azure Portal.
+
+    ![Resources created on Onpremises's resource group](./03.png)
+
+- Once deployed, connect to the Linux virtual machine using Azure Bastion. Once you login, the *install_arc_agent.sh* script execution will start and will onboard the VM to Azure Arc.
+
+    ![Connect with Azure Bastion 01](./04.png)
+
+    ![Connect with Azure Bastion 02](./05.png)
+
+    ![Arc onboarding script 01](./06.png)
+
+    ![Arc onboarding script 02](./07.png)
+
+- The Azure Arc-enabled server will be shown in the Azure resource group as a new resource.
+
+    ![Azure Arc-enabled server on Azure's resource group](./08.png)
+
+## Connectivity test
+
+- Now that all resources are deployed in both resources groups, you can verify that there is a private connection from the Azure Arc-enabled server to the Azure SQL database, this connection will go through the VPN gateways that connect the two VNETs and the traffic will continue over the Microsoft's network to reach the PaaS service on its private IP address since Azure Private link has been enabled.
```

Comment:
```suggestion
- Now that all resources are deployed in both resources groups, you can verify that there is a private connection from the Azure Arc-enabled server to the Azure SQL database, this connection will go through the VPN gateways that connect the two VNETs and the network traffic will continue over the Microsoft's network to reach the PaaS service on its private IP address since Azure Private link has been enabled.
```